samples/immutable-messages/Core_9/Receiver/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.ImmutableMessages.UsingInterfaces.Receiver";
+        var endpointConfiguration = new EndpointConfiguration("Samples.ImmutableMessages.UsingInterfaces.Receiver");
+        endpointConfiguration.UseSerialization<NewtonsoftJsonSerializer>();
```

Comment:
Would there be any value in switching this to SystemJsonSerializer just to lose the reference to Newtonsoft?